src/pharmpy/model/external/nonmem/dataset/_columns.py

```diff
@@ -0,0 +1,11 @@
+from pharmpy.deps import pandas as pd
+
+
+def idcol(df: pd.DataFrame) -> str | None:
+    columns = df.columns
+    if 'ID' in columns:
+        return 'ID'
+    elif 'L1' in columns:
+        return 'L1'
+    else:
+        return None
```

Comment:
A little to little to keep in separate file? Since it is related to how nmtran sees the dataset perhaps it could be placed there.

Reply:
I think there is other machinery that parses `$INPUT` to get the "true" names of the columns. It might be possible to write `$INPUT ID=MYID`, but that should be handled (not entirely sure without testing though)